starter_code/hangman.js

```diff
@@ -1,37 +1,70 @@
 let hangman;
+let hangmanCanvas;
+
+const minimumLetterRange = 65
+const maximumLetterRange = 90
+
+class Hangman {
+  constructor() {
+    this.words = ['Artanis', 'Batman', 'Luis']
+    this.secretWord = ''
+    this.guessedLetter = ''
+    this.letters = []
+    this.errorsLeft = 10
+  }
+  getWord() {
+    let randomIndex = Math.floor(Math.random() * this.words.length)
+    let secretWord = this.words[randomIndex]
+    return secretWord
+  }
+  checkIfLetter(keyCode) {
+    if (keyCode >= 65 && keyCode <= 90) {
+      return true
+    } else {
+      return false
+    }
+  }
+  checkClickedLetters(key) {
+    if (this.letters.includes(key) == false) {
+      return true
+    } else {
+      return false
+    }
+  }
+  addCorrectLetter(i) {
+    //addCorrectLetter. Adds to the guessedLetter variable the letter 
+    //that was pressed. Also, it should check if the user wins.
+    //pressedKey.push(element)
+    /*De acuerdo al ejercicio, me pide que agregue la letra que fue
+    presionada a guessedLetter y considero que eso es lo que 
+    estoy haciendo aquí pues sumo guessedLetter mas la letra que
+    fue presionada, es decir i; sin embargo, Jazmine no me lo marca.
+    Ahora, en Jazmin dice que debo de recibir un número y que como
```

Comment:
Un buen approach para este metodo era traer la var de secretWord para poder manipularla, y asignarla a una nueva variable  separandola con split(), despues le asignabamos a this.guessedLetter cada una d las letras separadas en mayuscula en caso de que el usuario escriba mayuscua o minuscula y ambos funcionen.

> The split() method turns a String into an array of strings, by separating the string at each instance of a specified separator string.

`let arrStrings = this.secretWord.split("")`
   ` this.guessedLetter += arrStrings[i].toUpperCase()`